docs/preview/features/writing-different-telemetry-types.md

```diff
@@ -64,7 +64,7 @@ durationMeasurement.Start();
 var startTime = DateTimeOffset.UtcNow;
 
 logger.LogBlobStorageDependency(accountName: "multimedia", containerName: "images", isSuccessful: true, startTime, durationMeasurement.Elapsed);
-// Output: "Dependency Azure blob multimedia named images in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
+// Output: "Dependency DependencyLogTarget {DependencyType: Azure blob multimedia named images in 00:00:00.2521801 at 03/23/2020 09:56:31 +00:00 (Successful: True - Context: )"
```

Comment:
Q: do we need to update the docs too? As in: should we change the log outputs in the https://observability.arcus-azure.net/features/writing-different-telemetry-types to this structured way?

Reply:
Yes please!

Reply:
But the current change doesn't look correct though

Reply:
Yes, I know, I was starting but stopped 😅 since I didn't know if I should change these too. It can be changed though the output template in Serilog and wasn't sure if it's a good idea to have a fixed representation of this here; because the way it's set up, you can still have the same output if you use the general `ToString` on our log models (and not use this Serilog decomposing formatting).

Reply:
Can you update the docs please?

Reply:
Done!